envoy-control-core/src/main/resources/lua/egress_service_tags.lua

```diff
@@ -0,0 +1,27 @@
+function envoy_on_request(handle)
+    rejectServiceTagDuplicatingAutoServiceTag(handle)
```

Comment:
This script looks strange now, but the plan is to add here a code related to upcoming feature: reading `service-tag-preference` header and put it to metadata for use by feature [metadata_fallback_policy](https://www.envoyproxy.io/docs/envoy/v1.24.2/api-v3/config/cluster/v3/cluster.proto#envoy-v3-api-field-config-cluster-v3-cluster-lbsubsetconfig-metadata-fallback-policy)